Tests/UnitTests/Plugins/Sycl/Seeding/CommandLineArguments.cpp

```diff
@@ -0,0 +1,61 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "CommandLineArguments.h"
+
+#include <fstream>
+#include <iostream>
+
+#include <boost/program_options.hpp>
+#include <boost/type_erasure/any_cast.hpp>
+
+namespace po = boost::program_options;
+
+namespace Acts {
+namespace Sycl {
+extern void listPlatforms();
```

Comment:
Wait... What's this? Why didn't you just include the appropriate header file from the plugin? Using `extern` here seems unnecessary anyway... :confused: